sdk/sdk-common/src/protocols/interfaces/MakerPoolId.ts

```diff
@@ -0,0 +1,12 @@
+import { ProtocolName } from './ProtocolName'
+import { IPoolId } from './IPoolId'
+
+// TODO: temporary interface so FE can create this data types without talking to a service
+export interface MakerPoolId extends IPoolId {
+  protocol: ProtocolName.Maker
+  id: string
```

Comment:
how about protocol & ilk?

Reply:
Sure! I'm happy changing it, I wasn't sure what was needed for Maker